File: src/main/java/com/EmosewaPixel/pixellib/proxy/IModProxy.java
package com.EmosewaPixel.pixellib.proxy;

import net.minecraftforge.fml.event.lifecycle.InterModEnqueueEvent;
import net.minecraftforge.fml.event.lifecycle.InterModProcessEvent;

//This is an interface used for making Mod Proxies
public interface IModProxy {
    void init();

    void enque(InterModEnqueueEvent e);

    void process(InterModProcessEvent e);
}

File: src/main/java/com/emosewapixel/pixellib/proxy/IModProxy.kt
package com.emosewapixel.pixellib.proxy

import net.minecraftforge.fml.event.lifecycle.InterModEnqueueEvent
import net.minecraftforge.fml.event.lifecycle.InterModProcessEvent

//This is an interface used for making Mod Proxies
interface IModProxy {
    fun init()

    fun enque(e: InterModEnqueueEvent)

    fun process(e: InterModProcessEvent)
}